### PR TITLE
Replace custom colours and colour usecases with brand configuration.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "o-assets": ">=0.4.1 <4",
     "o-viewport": ">=2.2.0 <4",
     "o-icons": ">=4.4.2 <6",
-    "o-spacing": "^2.0.0"
+    "o-spacing": "^2.0.0",
+    "o-brand": "^3.2.5"
```
Comment (JakeChampion) [inline bower.json]: I think we should add this to MIGRATION.md

File: main.scss [hunk 2]
```diff
@@ -75,14 +75,7 @@
 		position: relative;
 		border-top: 1px solid;
 		border-bottom: 1px solid;
-
-		// Hack to increase sepcificity to override overflow
-		// set on class used on the same element preventing use of !important.
-		// This Sass selector duplicates the class, creating:
-		// .o-footer__external-link.o-footer__external-link
-		&#{&} {
```
Comment (JakeChampion) [inline main.scss]: What a naughty hack

File: main.scss [hunk 3]
```diff
@@ -143,22 +136,12 @@
 	}
 
 	@if $martix {
```
Comment (JakeChampion) [inline main.scss]: typo?

File: bower.json (hunk 1 above)
Comment (notlee) [reply bower.json]: I'll update all our MIGRATION.md when we do the final rollout at the same time :)

File: main.scss (hunk 2 above)
Comment (notlee) [reply main.scss]: I also don't quite get what it was for... or how it would prevent `!important`...

File: main.scss (hunk 3 above)
Comment (notlee) [reply main.scss]: YES!

Comment (notlee) [reply main.scss]: Which effects the public interface. Good catch

**Comment by origamiserviceuser** [comment]
o-footer bundle size difference from 6.1.4 to 1ab677a9019a11e01a1579fa5f3df5b3d82cba88
No significant bundle size differences found.

**Pull request merged**